cmd/clock_generic.go

```diff
@@ -0,0 +1,13 @@
+// +build !test
+
+package cmd
+
+import "github.com/letsencrypt/boulder/Godeps/_workspace/src/github.com/jmhodges/clock"
+
+// Clock functions similarly to clock.Default(), but the returned value can be
+// changed using the FAKECLOCK environment variable in test environments.
```

Comment:
nit: this comment is still somewhat confusing when viewed on its own, perhaps adding "if the 'test' build flag is set" would help...?